firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -1972,6 +1974,22 @@ class BrowserViewController: UIViewController,
         presentSheetWith(viewModel: viewModel, on: self, from: button)
     }
 
+    func presentNewTabLongPressActionSheet(from view: UIView) {
+        let oneTapNewTabEnabled = featureFlags.isFeatureEnabled(.toolbarOneTapNewTab, checking: .buildOnly)
+        guard presentedViewController == nil, oneTapNewTabEnabled else { return }
+
+        let actions = getNewTabLongPressActions()
+
+        let shouldSuppress = UIDevice.current.userInterfaceIdiom != .pad
```

Comment:
should we just check iPad or shouldUseiPad? How does this look in multitasking?

Reply:
Was going for consistency with what's done for [long pressing the address bar](https://github.com/mozilla-mobile/firefox-ios/blob/be3fe148af5164b5120f3dfc424295b3128e1ae0/firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift#L1961) which seems to always present in popover for iPad (regardless of multitasking), however the [refresh button long press action](https://github.com/mozilla-mobile/firefox-ios/blob/be3fe148af5164b5120f3dfc424295b3128e1ae0/firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift#L1978) does consider multitasking / sizeclass (by way of checking if `TopTabsViewController` is nil)

Will check with design.

Reply:
thanks!

Reply:
Update: It was suggested that the presentation style should be based on size class, so anywhere you see the bottom navigation toolbar, the long press action will result in an action sheet instead of a popover